plugin/documents.py

```diff
@@ -917,6 +923,27 @@ def _update_stored_selection_async(self) -> Tuple[Optional[sublime.Region], bool
         self._stored_selection = selection
         return changed_first_region, True
 
+    def _format_on_paste(self) -> None:
+        self.purge_changes_async()
+        clipboard_text = sublime.get_clipboard()
+        sel = self.view.sel()
+        split_clipboard_text = clipboard_text.split('\n')
+        multi_cursor_paste = len(split_clipboard_text) == len(sel) > 1
```

Comment:
I wrote this line :-)
I thought that it would be more readable... this line is two comparisons with two numbers each, like you would write in maths. It does the same as `len(split_clipboard_text) == len(sel) and len(sel) > 1`, but with `len(sel)` only evaluated a single time.

Reply:
I see. So it's a Python `chained comparison` feature ([ref](https://stackoverflow.com/questions/57400620/python-comparison-operator-precedence)). Sorry but I don't like it. It's not clear from the code if it's intentional or it's just a bug in the code and all that just for optimization that doesn't even matter in this case :)

Reply:
I agree that the optimization doesn't really matter here, but it's much more readable than the long form, no?
What would it make more clear that it's not a bug? We could rename the variable to `has_multiple_selections_and_the_same_number_of_insertions`...
Do we need some kind of contribution guidelines with Python features which are allowed or are not allowed to be used? Otherwise we must assume that every line can be a bug....

Reply:
I suppose it's just a "me" issue and you can ignore this suggestion but I just really don't like this Python feature.

Probably because it looks just like a series of comparisons but behaves differently than any other language I know and use.

`len(sel) > 1 and len(split_clipboard_text) == len(sel)` would look so much cleaner to **me**.